pylatex/section.py

```diff
@@ -25,7 +25,7 @@ class Section(Container):
     #: subclasses will also have the new default.
     numbering = True
 
-    def __init__(self, title, numbering=None, *, label=True, **kwargs):
+    def __init__(self, title, short=None, numbering=None, *, label=True, **kwargs):
```

Comment:
please put the argument after the `*`, so that it's not a breaking change in the API. Also please update the docstring to explain what this new option does.